eZ/Publish/Core/Persistence/Legacy/Content/Handler.php

```diff
@@ -275,6 +283,14 @@ public function createDraftFromVersion($contentId, $srcVersion, $userId, ?string
         // Clone fields from previous version and append them to the new one
         $this->fieldHandler->createExistingFieldsInNewVersion($content);
 
+        // Persist virtual fields
+        $contentType = $this->contentTypeHandler->load($content->versionInfo->contentInfo->contentTypeId);
+        $this->fieldHandler->updateFields($content, new UpdateStruct([
+            'initialLanguageId' => $this->languageHandler->loadByLanguageCode(
+                $content->versionInfo->initialLanguageCode
+            )->id,
+        ]), $contentType);
```

Comment:
Please update PHPDoc with  `@throws \eZ\Publish\API\Repository\Exceptions\NotFoundException` exception thrown from this new code.